CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Log Change Log
 
-## 1.0.5 under development
+## 1.1.0 under development
```

Comment:
When raising the PHP version, isn't this a major release?

Reply:
Recently discussed in private chat and decided that this can be a minor.

Reply:
But PR also has changes in public API and this is major changes. Seems, it is should be 2.0.0 version.

Reply:
Yes.